core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/hybrid/HybridJDBCConstants.java

```diff
@@ -107,6 +128,12 @@
             + ".UM_TENANT_ID=? AND UM_HYBRID_USER_ROLE.UM_DOMAIN_ID=(SELECT UM_DOMAIN_ID FROM UM_DOMAIN WHERE "
             + "UM_TENANT_ID=? AND UM_DOMAIN_NAME=?)";
 
+    public static final String GET_INTERNAL_ROLE_LIST_OF_GROUPS_SQL = "SELECT UM_GROUP_NAME, UM_ROLE_NAME FROM "
+            + "UM_HYBRID_GROUP_ROLE, UM_HYBRID_ROLE WHERE UM_GROUP_NAME IN (?) AND UM_HYBRID_GROUP_ROLE"
```

Comment:
Why the table name is UM_HYBRID_GROUP_ROLE? We don't have HYBRID_GROUP... prefer UM_GROUP_ROLE

Reply:
We do have a table called UM_HYBRID_USER_ROLE to maintain the role and user association. So a similar name was used for GROUP as well to maintain consistency.